worlds/noita/Items.py

```diff
@@ -0,0 +1,147 @@
+import itertools
+from typing import Dict, NamedTuple, Optional, List, Set
+from BaseClasses import Item, ItemClassification, MultiWorld
+from . import Options
+
+
+class ItemData(NamedTuple):
+    code: Optional[int]
+    group: str
+    classification: ItemClassification = ItemClassification.progression
+    required_num: int = 0
+
+
+class NoitaItem(Item):
+    game: str = "Noita"
+
+
+def create_item(player: int, name: str) -> Item:
+    item_data = item_table[name]
+    return NoitaItem(name, item_data.classification, item_data.code, player)
+
+
+def create_all_items(world: MultiWorld, player: int) -> None:
+    sum_locations = len(world.get_unfilled_locations(player))
+
+    # Generate fixed item pool, these are items with a specific fixed quantity that must be added
+    itempool: List = []
+    for item_name, count in required_items.items():
+        itempool += [item_name] * count
+
+    # Generate items based on victory condition settings (Orbs)
+    vic = world.victory_condition[player].value
+    orb_count = 0
+
+    if vic == Options.VictoryCondition.option_pure_ending:
+        orb_count = 11
+    if vic == Options.VictoryCondition.option_peaceful_ending:
```

Comment:
No need to check this with a separate `if` if the previous statement was true. :P

```suggestion
    elif vic == Options.VictoryCondition.option_peaceful_ending:
        orb_count = 33
```